### PR TITLE
Sounds 

### PR DESCRIPTION
**Pull request by ajslaterhb2000**
branch name is a bit off, added place holder sounds for tag, item pick up and jump. Also have a place holder background track

### CODE REVIEW

File: include/client/graphics/Player.h [hunk 1]
```diff
@@ -49,6 +53,7 @@ class Player : public GameThing, InputListener {
     if (fx_land) delete fx_land;
     if (fx_item) delete fx_item;
     if (fx_tag) delete fx_tag;
+    if (sfx_jump) delete sfx_jump;
```
Comment (yesean) [inline include/client/graphics/Player.h:56]: can you delete the other sfx resources too?

File: include/client/graphics/Scene.h [hunk 2]
```diff
@@ -165,6 +166,8 @@ class Scene {
 
     // the default scene graph already has one node named "world."
     node["world"] = new Node("world");
+    music = new Music();
```
Comment (yesean) [inline include/client/graphics/Scene.h:169]: can you delete the music resource too?

**Review by yesean — CHANGES_REQUESTED** [review]

File: include/client/graphics/Scene.h (hunk 2 above)
Comment (ajslaterhb2000) [reply include/client/graphics/Scene.h:169]: added in newest commit

File: include/client/graphics/Player.h (hunk 1 above)
Comment (ajslaterhb2000) [reply include/client/graphics/Player.h:56]: added in newest commit

**Review by yesean — APPROVED** [review]
lgtm

**Pull request merged**